src/main/java/com/godev/linkhubservice/domain/vo/CreatePageRequest.java

```diff
@@ -36,19 +36,19 @@ public class CreatePageRequest {
     @Length(min = 4, max = 100, message = TITLE_LENGTH_ERROR)
     private String title;
 
-    @Schema(name = "description", defaultValue = "Link hub page description.", description = "A description of the page")
+    @Schema(name = "description", defaultValue = "Link Hub page description.", description = "A description of the page")
     @Length(min = 20, max = 200, message = DESCRIPTION_LENGTH_ERROR)
     private String description;
 
-    @Schema(name = "photo", defaultValue = "Link hub page photo.", description = "A photo of the page")
+    @Schema(name = "photo", defaultValue = "default.png", description = "A photo of the page")
```

Comment:
Evaluate to use imgur url as default value

Reply:
resolved